backend/TreeOfAKind.Application/Query/Trees/GetTreeStatistics/TreeStatisticsDto.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using TreeOfAKind.Domain.Trees.People;
+
+namespace TreeOfAKind.Application.Query.Trees.GetTreeStatistics
+{
+    public record TreeStatisticsDto
+    {
+        public int TotalNumberOfPeople { get; }
+        public double AverageLifespanInDays { get; }
+        public Dictionary<Gender, int> NumberOfPeopleOfEachGender { get; }
+        public int NumberOfLivingPeople { get; }
+        public int NumberOfDeceasedPeople { get; }
+        public double AverageNumberOfChildren { get; }
+        public int NumberOfMarriedPeople { get; }
+        public int NumberOfSinglePeople { get; }
+
+        public TreeStatisticsDto(int totalNumberOfPeople, double averageLifespanInDays,
+            Dictionary<Gender, int> numberOfPeopleOfEachGender, int numberOfLivingPeople, int numberOfDeceasedPeople,
+            double averageNumberOfChildren, int numberOfMarriedPeople, int numberOfSinglePeople)
+        {
+            TotalNumberOfPeople = totalNumberOfPeople;
+            AverageLifespanInDays = averageLifespanInDays;
+            NumberOfPeopleOfEachGender = numberOfPeopleOfEachGender ?? new Dictionary<Gender, int>();
```

Comment:
Probably would be more convenient if it would return 0's for every type of gender, same goes for genders not included in LINQ.

Reply:
Actually doesn't matter